pkg/registry/apis/provisioning/types.go

```diff
@@ -4,8 +4,10 @@ import (
 	"context"
 
 	"github.com/grafana/grafana/pkg/registry/apis/provisioning/repository"
+	"github.com/grafana/grafana/pkg/registry/apis/provisioning/webhook"
 )
 
-type RepoGetter interface {
+type Getter interface {
 	GetRepository(ctx context.Context, name string) (repository.Repository, error)
+	GetWebhook(ctx context.Context, name string) (webhook.Webhook, error)
```

Comment:
I don't see how this helps us much -- IIUC, the biggest challenge is splitting the webhook behavior from both parsing+processing into just parsing jobs.  

Something like https://github.com/grafana/grafana/pull/97544